meson.build

```diff
@@ -1013,17 +1013,8 @@ endif
 
 #####################################################################
 
-libatomic = []
-if not cc.links('''#include <stdint.h>
-                   int main() {
-                     char i;
-                     __atomic_exchange_1(&i, 1, 0);
-                     return 1;
-                   }''',
-                name : 'Atomic builtin requires -latomic')
-        libatomic = cc.find_library('atomic')
-endif
 threads = dependency('threads')
+libatomic = cc.find_library('atomic')
```

Comment:
We need required: false as it's not available with compiler-rt but it doesn't matter that it's absent.

Reply:
Ah, indeed. I looked at the docs, and the docs say "If **set** `true`, Meson will abort with an error if the library could not" (emphasis mine), and I assumed that the default is `false`.